pkgs/tools/inputmethods/input-remapper/default.nix

```diff
@@ -81,9 +81,14 @@ in
     psutil
   ];
 
-  doCheck = withDoCheck;
+  # buildPythonApplication maps nativeCheckInputs to nativeInstallCheckInputs.
+  nativeCheckInputs = [
+    udevCheckHook
+    versionCheckHook
+  ]
+  ++ lib.optionals withDoCheck [ psutil ];
```

Comment:
I shouldn't have named that `withDoCheck` 😅

Reply:
What new name would you suggest? We could rename it if you like. With `lib.oldestSupportedReleaseIsAtLeast` we could set a future warning and backport the whole thing compatibly.

Reply:
maybe `withTests`? idk if it's worth renaming though, definitely not blocking feedback.